CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 0.80.0 / 2025-01-30
+
+* [CHANGE] Add more API validations to the ScrapeConfig CRD. #7231 #7220 #7215
+* [FEATURE] Add `PrometheusTopologySharding` feature gate. #7233
```

Comment:
may be this is not needed now as feature not implemented IIUC

Reply:
correct, I would omit for now.